before-deploy.sh

```diff
@@ -10,4 +10,5 @@ ln -s /usr/bin/FileCheck-8 $name/bin/FileCheck
 cp clang/tools/extra/clang-tidy/*.h $name/include
 cp clang/LICENSE.TXT $name/clang-LICENSE.TXT
 ln -s /usr/include/clang $name/lib/clang
+cp clang/build/lib/libclangTidyMain.a $name/lib
```

Comment:
I only changed it to build clangTidyMain.a instead of clang-tidy, without changing the dependencies. So if clang-tidy is statically linked, clangTidyMain.a should also be.